src/middleware/requestLogger.ts

```
@@ -0,0 +1,104 @@
+ *   - correlationId (x-request-id header or auto-generated UUID)
+ *   - standard HTTP fields (method, path, statusCode, durationMs, userAgent)
+ */
+export const requestLogger = (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+): void => {
+  const startHr = process.hrtime.bigint();
+  const timestamp = microTimestamp();
+  const correlationId =
+    (req.headers["x-request-id"] as string | undefined) ?? crypto.randomUUID();
```

ESLint

[error] Delete `⏎···`

(prettier/prettier)